hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -2206,6 +2264,34 @@ public void testCacheMerkleTreeConfig() {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testCacheWithoutMerkleTreeConfig() {
+        Config cfg = new Config();
+        MerkleTreeConfig actual = cfg.getCacheConfig("testCacheWithoutMerkleTreeConfig")
+                .getMerkleTreeConfig();
+
+        MerkleTreeConfig expected = getNewConfigViaXMLGenerator(cfg)
+                .getCacheConfig("testCacheWithoutMerkleTreeConfig").getMerkleTreeConfig();
+
+        assertEquals(expected, actual);
+    }
+
```

Comment:
Minor: Missing counterpart of the `testMapWithEnabledMerkleTreeConfig` for the `CacheConfig`

Reply:
there is already `testCacheMerkleTreeConfig` that tests this case